DESCRIPTION

```diff
@@ -17,5 +17,17 @@ RoxygenNote: 7.3.2
 Imports: 
     ggplot2,
     survival
+Suggests:
+  adjustedCurves,
+  arsenal,
+  data.table,
+  DiagrammeR,
+  dplyr,
+  gt,
+  knitr,
+  pammtools,
+  rmarkdown,
+  survminer
```

Comment:
Any package used in the vignettes and not already in `Imports` needs to be in `Suggests`